cfgov/unprocessed/js/modules/util/ajax-request.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+
+/**
+* Initiates an ajax request when passed a type, url,
+* and an optional opts object.
+*
+* @param {string} type request type (GET, POST)
+* @param {string} url request url
+* @param {object} opts object containing optional
```

Comment:
Should be fixed in https://github.com/cfpb/cfgov-refresh/pull/2985/commits/fe4fce08f81e40367e318dc7511e6f1fd3ad2ced